src/axom/primal/operators/detail/winding_number_impl.hpp

```diff
@@ -109,11 +114,21 @@ double convex_endpoint_winding_number(const Point<T, 2>& q,
   // Need to find vectors that subtend the entire curve.
   //   We must ignore duplicate nodes
   for(idx = 0; idx <= ord; ++idx)
-    if(squared_distance(q, c[idx]) > edge_tol_sq) break;
+  {
+    if(squared_distance(q, c[idx]) > edge_tol_sq)
+    {
+      break;
+    }
+  }
   Vector<T, 2> V1(q, c[idx]);
 
   for(idx = ord; idx >= 0; --idx)
-    if(squared_distance(q, c[idx]) > edge_tol_sq) break;
+  {
+    if(squared_distance(q, c[idx]) > edge_tol_sq)
+    {
+      break;
+    }
+  }
```

Comment:
Note: There is an bug in the `clang-tidy` when adding braces around nested if/for/while statements where it will add an opening brace for each item, but will only add a single closing brace. I had to manually add additional closing braces.

I was using `clang-tidy@10.0.1` -- not sure if this has been fixed in later versions.